packages/core/src/button/Button.tsx

```diff
@@ -38,6 +38,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
       onBlur,
       onClick,
       role: roleProp,
+      type: typeProp,
```

Comment:
```suggestion
      type = "button",
```
There's no point renaming the prop here and we should try and keep defaults with the props.